tests/transforms/test_convert_rois.py

```diff
@@ -138,6 +220,5 @@ def test_binarize_and_convert_rois(s2p_stat_fixture, ophys_movie_fixture,
 
     with open(output_path) as open_output:
         rois = json.load(open_output)
-        # assert all rois were written
-        assert len(rois) == len(stat_fixure_params['masks'])
```

Comment:
I removed this assertion because ROIs are now removed in one of the test cases so it's no longer relevant.

Reply:
Ok, so what is this testing now? It appears like a "smoke test" to me. I think you should be able to provide the expected values, to turn this into more of an integration/unit test.